### DISCUSSION TITLE
Can we plot inverted graph with points only showing colors

### DISCUSSION THREAD

**Discussion by vibhoothi** [colour-science/colour#1092] [Q&A] [answered]
Hi Team, 

This library is amazing, 

I was wondering whether can we plot the graph where we show only the points with real colors and rest being blank/white, 
For example, from ITU, https://www.itu.int/dms_pub/itu-r/opb/rep/R-REP-BT.2246-5-2015-PDF-E.pdf#page=39

<img width="327" alt="image" src="https://user-images.githubusercontent.com/10833993/210456199-fff2cf55-1270-4b64-817a-5b6f7da8315e.png">

I tried to do a quick search online, but not sure whether we could do it, apologies if this is asked previously,

**Comment by thomasmansencal** [comment, marked as answer]
Hi @vibhoothi,

This should be doable yes!

```python
RGB = np.random.random([1280, 720, 3])[::10, :]

colour.plotting.plot_RGB_chromaticities_in_chromaticity_diagram_CIE1931(
    RGB,
    colourspaces=["DCI-P3"],
    plot_kwargs={"color": "k"},
    scatter_kwargs={"s": 2},
    show_diagram_colours=False,
    legend=False)
```

![image](https://user-images.githubusercontent.com/99779/210524637-3d915cca-2d2f-46d1-ba67-fd78b1be5794.png)

**Reply by vibhoothi** [reply]
Thanks for the fast reply, 
Amazing this works for my use-case
